.github/workflows/test-go.yaml

```diff
@@ -129,13 +129,13 @@ jobs:
         flags: backend
 
     - name: Generate summary of errors
-      if: github.event.schedule == '0 4 * * *' && failure()
+      if: failure()
```

Comment:
nice, the only thing to improve in a future iteration is that the summary will be printed in this step:

<img width="919" alt="image" src="https://github.com/fleetdm/fleet/assets/4419992/047a265e-f841-4cd8-89f0-f322284ec22f">

but GitHub opens the failed step by default

Reply:
Agreed, it could be made much better